pkg/pillar/cmd/client/client.go

```diff
@@ -450,40 +508,40 @@ func myPost(zedcloudCtx zedcloud.ZedCloudContext, tlsConfig *tls.Config,
 		log.Errorf("%s StatusConflict\n", requrl)
 		// Retry until fixed
 		log.Errorf("%s\n", string(contents))
-		return false, resp, contents
-	case http.StatusNotModified:
+		return false, resp, remExtStatus, contents
+	case http.StatusNotFound, http.StatusUnauthorized, http.StatusNotModified:
 		// Caller needs to handle
-		return false, resp, contents
+		return false, resp, remExtStatus, contents
 	default:
 		log.Errorf("%s statuscode %d %s\n",
 			requrl, resp.StatusCode,
 			http.StatusText(resp.StatusCode))
 		log.Errorf("%s\n", string(contents))
-		return false, resp, contents
+		return false, resp, remExtStatus, contents
 	}
 
 	contentType := resp.Header.Get("Content-Type")
 	if contentType == "" {
 		log.Errorf("%s no content-type\n", requrl)
-		return false, resp, contents
+		return false, resp, remExtStatus, contents
 	}
 	mimeType, _, err := mime.ParseMediaType(contentType)
 	if err != nil {
 		log.Errorf("%s ParseMediaType failed %v\n", requrl, err)
-		return false, resp, contents
+		return false, resp, remExtStatus, contents
 	}
 	switch mimeType {
 	case "application/x-proto-binary", "application/json", "text/plain":
 		log.Debugf("Received reply %s\n", string(contents))
 	default:
 		log.Errorln("Incorrect Content-Type " + mimeType)
-		return false, resp, contents
+		return false, resp, remExtStatus, contents
 	}
-	return true, resp, contents
+	return true, resp, remExtStatus, contents
 }
 
 // Returns true when done; false when retry
```

Comment:
Please update comment to specify what the second return value is.

Reply:
ok